RELEASING.md

```diff
@@ -2,7 +2,7 @@
 1. Start a git-flow release/x.y.z
 1. Create a CHANGELOG.latest.md with the changes for the current 
 version (to be used by Fastlane for the github release notes)
-1. Update the version number by running `fastlane bump_and_update_changelog version:x.y.z`
+1. Update the version number by running `bundle exec fastlane bump_and_update_changelog version:x.y.z`
```

Comment:
whenever you use `fastlane` directly, you get a warning saying that you should call it through `bundle exec` to speed up fastlane's startup time 🤷